primitive-types/impls/rlp/Cargo.toml

```
@@ -7,4 +7,4 @@ homepage = "https://github.com/paritytech/parity-common"
 description = "RLP serialization support for uint and fixed hash."
 
 [dependencies]
-rlp = { version = "0.3", path = "../../../rlp", default-features = false }
+rlp = { version = "0.4", path = "../../../rlp", default-features = false }
```

we don't need these `default-features` anymore
```suggestion
rlp = { version = "0.4", path = "../../../rlp" }
```